### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by obed300**
In this branch;

I added the work section and  the about myself session.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,12 +60,207 @@ <h1 class="item">I’m Obed Bamfo Glad to see you!</h1>
...
+                    </a>
+                    <li><a href="#">
+                        <img src='images/MI.svg' alt="mill" >
+                    </a></li>  
+                </ul> 
+                <div class="resume">
+                    <p class="get">Get my resume</p>
+                </div>
+                <div class="languages">
+                    <p class="write-in">Languages</p>
```
Comment (Reem-lab) [inline index.html:227]: - [ ] I think this languages and skills and frameworks have more font weight so kindly add some font weight to make your portfolio warm to see and smooth and similar to the Figma design as much as possible 💯💪🏻✅

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @obed300, 👋🏻👋🏻

Good job so far! 💥💥💯💪🏻💪🏻
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the design✅
📌 Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -1,93 +1,757 @@
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background: #fff;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+}
```
Comment (levy002) [inline style.css:190]: - [ ] As requested by the previous reviewer kindly make sure to style correctly the `see project` button in order to look like the provided Figma design. :wink: 
 
   ### PLease check out my suggestions:

```suggestion
.see {
  background: #fff;
  border: 1px solid #396df2;
  border-radius: 7px;
  padding: 7px 5px;
  color: #396df2;
  font-family: 'Poppins', sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 15px;
  line-height: 24px;
}
```

File: style.css [hunk 3]
```diff
@@ -1,93 +1,757 @@
...
-  flex-grow: 0;
 }
 
-.connect {
+.connect-two {
+  display: flex;
   margin-left: 24px;
   margin-top: 12px;
+  width: 120px;
   height: 24px;
```
Comment (levy002) [inline style.css:742]: - [ ] As requested by the previous reviewer, kindly make sure to align correctly the `LET'S CONNECT` title.

  ### hint: Please remove `width` everything will work well.

**Comment by obed300** [comment]
Please I have done the changes as you requested. Please check it and let me
know here else I will be exceeding the number of reviews I have on this
project

File: index.html [hunk 4]
```diff
@@ -8,63 +8,200 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>
 </head>
 <body>
 
 
         <div class="main-container">
-            <header class="header-section">
```
Comment (Tadesse-Alemayehu) [inline index.html:16]: - [ ]  Good job making the changes @obed300, I have suggestion needs to be in place. You over all layout seems to be out of place. Please consider fixing the width and the height issues in your design.

### Your current design. 
[Screencast from 28-09-2022  3:34:11 ከሰዓት.webm](https://user-images.githubusercontent.com/69077061/192780973-91bfc5a1-8c30-45e8-bc86-d15700ceadac.webm)

**Review by Tadesse-Alemayehu — APPROVED** [review]
Hi @obed300 

Your project is complete!  Also, I have added strong suggestion to make, Please consider adding those.
Congratulations! 🎉 

approve_happy3

## Highlights
- Making changes from the previous code reviewer  :heavy_check_mark: 
- Pushpin No Linters error :heavy_check_mark: 
- Pushpin Descriptive pull request :heavy_check_mark: 
- Pushpin Great working on the design :heavy_check_mark: 
- Pushpin Good commit messages  :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**